meta-facebook/yv35-cl/src/platform/plat_sensor_table.c

```diff
@@ -0,0 +1,214 @@
+	  0, 0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page,
+	  &isl69260_page_data_args[0], NULL, NULL },
+	{ SENSOR_NUM_TEMP_PVCCINFAON, TYPE_ISL69260, I2C_BUS5, PVCCINFAON_ADDR, VR_TEMP_CMD,
+	  DC_access, 0, 0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page,
+	  &isl69260_page_data_args[0], NULL, NULL },
+	{ SENSOR_NUM_TEMP_PVCCFA_EHV, TYPE_ISL69260, I2C_BUS5, PVCCFA_EHV_ADDR, VR_TEMP_CMD,
+	  DC_access, 0, 0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page,
+	  &isl69260_page_data_args[1], NULL, NULL },
+	{ SENSOR_NUM_TEMP_PVCCIN, TYPE_ISL69260, I2C_BUS5, PVCCIN_ADDR, VR_TEMP_CMD, DC_access, 0,
+	  0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page, &isl69260_page_data_args[0],
+	  NULL, NULL },
+	{ SENSOR_NUM_TEMP_PVCCFA_EHV_FIVRA, TYPE_ISL69260, I2C_BUS5, PVCCFA_EHV_FIVRA_ADDR,
+	  VR_TEMP_CMD, DC_access, 0, 0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page,
+	  &isl69260_page_data_args[1], NULL, NULL },
+
+	// VR power
+	{ SENSOR_NUM_PWR_PVCCD_HV, TYPE_ISL69260, I2C_BUS5, PVCCD_HV_ADDR, VR_PWR_CMD, DC_access, 0,
+	  0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page, &isl69260_page_data_args[0],
+	  NULL, NULL },
+	{ SENSOR_NUM_PWR_PVCCINFAON, TYPE_ISL69260, I2C_BUS5, PVCCINFAON_ADDR, VR_PWR_CMD,
+	  DC_access, 0, 0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page,
+	  &isl69260_page_data_args[0], NULL, NULL },
+	{ SENSOR_NUM_PWR_PVCCFA_EHV, TYPE_ISL69260, I2C_BUS5, PVCCFA_EHV_ADDR, VR_PWR_CMD,
+	  DC_access, 0, 0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page,
+	  &isl69260_page_data_args[1], NULL, NULL },
+	{ SENSOR_NUM_PWR_PVCCIN, TYPE_ISL69260, I2C_BUS5, PVCCIN_ADDR, VR_PWR_CMD, DC_access, 0, 0,
+	  0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page, &isl69260_page_data_args[0],
+	  NULL, NULL },
+	{ SENSOR_NUM_PWR_PVCCFA_EHV_FIVRA, TYPE_ISL69260, I2C_BUS5, PVCCFA_EHV_FIVRA_ADDR,
+	  VR_PWR_CMD, DC_access, 0, 0, 0, SENSOR_INIT_STATUS, NONE, NONE, isl69260_switch_page,
+	  &isl69260_page_data_args[1], NULL, NULL },
+
+	// ME
+	{ SENSOR_NUM_TEMP_PCH, TYPE_PCH, I2C_BUS3, PCH_ADDR, ME_SENSOR_NUM_TEMP_PCH, post_access, 0,
+	  0, 0, SENSOR_INIT_STATUS, NONE, NONE, NULL, NULL, NULL, NULL },
+
+	// HSC
+	{ SENSOR_NUM_TEMP_HSC, TYPE_HSC, I2C_BUS2, HSC_ADDR, HSC_TEMP_CMD, stby_access, 0, 0, 0,
+	  SENSOR_INIT_STATUS, NONE, NONE, NULL, NULL, NULL, NULL },
+	{ SENSOR_NUM_VOL_HSCIN, TYPE_HSC, I2C_BUS2, HSC_ADDR, HSC_VOL_CMD, stby_access, 0, 0, 0,
+	  SENSOR_INIT_STATUS, NONE, NONE, NULL, NULL, NULL, NULL },
+	{ SENSOR_NUM_CUR_HSCOUT, TYPE_HSC, I2C_BUS2, HSC_ADDR, HSC_CUR_CMD, stby_access, 0, 0, 0,
+	  SENSOR_INIT_STATUS, NONE, NONE, NULL, NULL, NULL, NULL },
+	{ SENSOR_NUM_PWR_HSCIN, TYPE_HSC, I2C_BUS2, HSC_ADDR, HSC_PWR_CMD, stby_access, 0, 0, 0,
+	  SENSOR_INIT_STATUS, NONE, NONE, NULL, NULL, NULL, NULL }
+};
+
+snr_cfg fix_class2_sensor_config_table[] = {
+	/* number,                  type,       port,      address,      offset,
```

Comment:
Are these tables placeholders for future work?  If so, we should add a TODO to indicate the remaining work here (and in the next two empty tables as well)

Reply:
We have fixed this and done the forced update.
Please review it. Thanks.